Doc/library/heapq.rst

```diff
@@ -346,19 +340,18 @@ last 0'th element you extracted.  This is especially useful in simulation
 contexts, where the tree holds all incoming events, and the "win" condition
 means the smallest scheduled time.  When an event schedules other events for
 execution, they are scheduled into the future, so they can easily go into the
-heap.  So, a heap is a good structure for implementing schedulers (this is what
-I used for my MIDI sequencer :-).
```

Comment:
It's a shame to 'sterilise' things by removing lines like this. If you feel strongly, perhaps rework to keep the example but remove the perpendicular pronoun.